src/main/fc/cli.c

```diff
@@ -124,10 +124,20 @@ extern uint8_t __config_end;
 
 
 static serialPort_t *cliPort;
+
+#ifdef STM32F1
+#define CLI_IN_BUFFER_SIZE 128
+#define CLI_OUT_BUFFER_SIZE 64
+#else
+// Space required to set array parameters
+#define CLI_IN_BUFFER_SIZE 256
+#define CLI_OUT_BUFFER_SIZE 256
```

Comment:
Weird:
```
#define CLI_IN_BUFFER_SIZE 256
#define CLI_OUT_BUFFER_SIZE 256
```
yields
```
# set dummy = 255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255
dummy set to 255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255,255
# 
```

whereas

```
#define CLI_IN_BUFFER_SIZE 256
#define CLI_OUT_BUFFER_SIZE 64
```
yields
```
# set dummy = 255,255,255,255,255,255,255,255,255,255,255,255,255,
Invalid value
Array length: 50

# 
```

Looks like big enough in **and** out buffers are required after all.